packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getPhonesFieldPreviewValue.ts

```diff
@@ -1,9 +1,27 @@
 import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import { DEFAULT_PHONE_CALLING_CODE } from '@/object-record/record-field/meta-types/input/components/PhonesFieldInput';
 import { FieldPhonesValue } from '@/object-record/record-field/types/FieldMetadata';
 import { getSettingsFieldTypeConfig } from '@/settings/data-model/utils/getSettingsFieldTypeConfig';
+import {
+  CountryCode,
+  getCountries,
+  getCountryCallingCode,
+} from 'libphonenumber-js';
 import { FieldMetadataType } from '~/generated-metadata/graphql';
 import { stripSimpleQuotesFromString } from '~/utils/string/stripSimpleQuotesFromString';
 
+const isStrCountryCodeGuard = (str: string): str is CountryCode => {
+  return getCountries().includes(str as CountryCode);
+};
+
+export const countryCodeToCallingCode = (countryCode: string): string => {
```

Comment:
```typescript
export const countryCodeToCallingCode = (countryCode: string): string => {
  if (!countryCode || !isStrCountryCodeGuard(countryCode)) {
    return `+${DEFAULT_PHONE_CALLING_CODE}`;
  }

  const callingCode = getCountryCallingCode(countryCode);

  return callingCode ? `+${callingCode}` : `+${DEFAULT_PHONE_CALLING_CODE}`;
};
```
Can we use if blocks in general? I think it's more readable this way

Reply:
If that's the prefered way, let's do it